src/Geometry.cpp

```diff
@@ -835,8 +824,51 @@ void Geometry::segmentize(Track* track) {
   /* Truncate the linked list for the LocalCoords */
   start.prune();
   end.prune();
+}
+
+
+/**
+ * @brief Initialize key and material ID vectors for lookup by FSR ID
+ * @detail This function initializes and sets reverse lookup vectors by FSR ID.
+ *      This is called after the FSRs have all been identified and allocated
+ *      during segmentation.
```

Comment:
can you just add a clarifying comment here in the ``@detail`` doxygen comment block that specifies which routine(s) must be called before this one, *i.e.*, ``TrackGenerator::generateTracks()`` or something even more specific